js/index.js

```diff
@@ -39,4 +39,79 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+// const imghead = document.querySelector("logo.png");
+// imghead.setAttribute('src', "img/logo.png");
+
+let headimg = document.getElementById("cta-img");
+headimg.setAttribute( "src", siteContent["cta"]["img-src"]);
+
+// const heading = document.querySelector("cta-img");
+// headimg.setAttribute('src',"cta-img");
+
+let navo = document.querySelectorAll("a");
+navo.forEach(function(item, index){
+item.textContent = siteContent["nav"][`nav-item-${index + 1}`];
+
+});
+
+
+
+
+let headtext = document.querySelector(".cta-text h1");
+headtext.textContent = siteContent["cta"]["h1"];
+
+
+let newButton = document.querySelector(".cta-text button");
```

Comment:
you're alternating between all-lowercase names for your variables like `headtext` and camelcase like here: camelcase names are much more preferred, but the most important thing is for you to stay consistent when choosing your names